packages/react/src/components/RadioTile/RadioTile.tsx

```diff
@@ -99,27 +164,27 @@ const RadioTile = React.forwardRef(function RadioTile(
 
 RadioTile.propTypes = {
```

Comment:
I took the liberty of updated these prop descriptions to better match the writing style of other components (e.g., `TileGroup`, `Select`, etc..). If you feel this is out of scope for this PR, I don't mind reverting these changes.